nnpdf_data/nnpdf_data/commondata/ATLAS_Z0J_8TEV/rawdata/hepdata/F2AV_README

```diff
@@ -0,0 +1,525 @@
+
```

Comment:
Hi @scarlehoff, thanks for the comments. I would keep it because it contains non-trivial information about the breakdown of the systematic uncertainties in the source files, which are used to build the covmat. I think it might be a good idea to keep this kind of information, also for future reference.

Reply:
Going forward we are removing _all_ data in the `rawdata` directory, see https://github.com/NNPDF/nnpdf/issues/2199

If there is some information that is important, summarize it and add it as a comment or module-level docstring to the `filter.py`.

Reply:
Ok, I'll summarise the relevant information in a module-level docstring.